### PR TITLE
Small edits of URIs section

### PR DESCRIPTION
**Pull request by lexaknyazev**
Split normative/non-normative parts of URIs section, disallow empty URIs in schema.

### CODE REVIEW

File: specification/2.0/schema/buffer.schema.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
             "type": "string",
             "description": "The uri of the buffer.",
             "format": "uriref",
+            "minLength": 1,
```
Comment (bghgary) [inline specification/2.0/schema/buffer.schema.json]: This is a schema change. I think we wanted to make sure the schema is locked?

Comment (lexaknyazev) [inline specification/2.0/schema/buffer.schema.json]: I think that such restriction was always implicit:
- `uri` is an _external_ reference;
- empty string URI points to the glTF file itself.

Following this logic, a valid glTF asset can't have empty string URIs.

If making such schema adjustment is impossible, we should document in the readme how to treat empty URIs.

Comment (bghgary) [inline specification/2.0/schema/buffer.schema.json]: CC @sbtron 

Our opinion is that this change isn't necessary. As you say an empty string URI points to the glTF file itself (which isn't valid), but you can also put a URI to some other random file (whether glTF or not) and that's also not valid. Clients must deal with this whether or not we allow empty URIs.

**Comment by lexaknyazev** [comment]
@bghgary 
I've reverted schema changes. Are you OK with readme edits?

**Comment by bghgary** [comment]
Yes.

**Pull request merged**